packages/core/src/tracing/instrumentation/messaging/kafkaJs.js

```diff
@@ -532,3 +535,11 @@ function removeInstanaHeadersFromMessage(message) {
     });
   }
 }
+
+// Note: This function can be removed as soon as we finish the Kafka header migration phase2.
+function logWarningForKafkaHeaderFormat() {
+  logger.warn(
+    '[Deprecation Warning] The configuration option for specifying the Kafka header format will be removed in the ' +
+      "next major release. The default header format will be 'string'."
```

Comment:
suggestion: Add the docs link for the kafka header migration.

Reply:
https://www.ibm.com/docs/en/instana-observability/current?topic=nodejs-collector-configuration#kafka-trace-correlation-headers